codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ValueEntitySourceGenerator.scala

```diff
@@ -86,6 +87,54 @@ object ValueEntitySourceGenerator {
         |""".stripMargin
   }
 
+  private[codegen] def handler(valueEntity: ModelBuilder.ValueEntity, service: ModelBuilder.EntityService): String = {
+    val stateType = valueEntity.state.fqn.name
+    val packageName = valueEntity.fqn.parent.scalaPackage
+    val valueEntityName = valueEntity.fqn.name
+    implicit val imports =
+      generateImports(
+        Seq(valueEntity.state.fqn) ++
+        service.commands.map(_.inputType) ++
+        service.commands.map(_.outputType),
+        valueEntity.fqn.parent.scalaPackage,
+        otherImports = Seq(
+          "com.akkaserverless.scalasdk.valueentity.CommandContext",
+          "com.akkaserverless.scalasdk.valueentity.ValueEntity",
+          "com.akkaserverless.scalasdk.impl.valueentity.ValueEntityHandler",
+          "com.akkaserverless.scalasdk.impl.valueentity.ValueEntityHandler.CommandHandlerNotFound"),
```

Comment:
This might be wrong. In View and Actions we used the corresponding javasdk exception instead of defining a new in scalasdk. It's not something seen by the user. In that we we don't need to catch-convert-throw.